superbench/benchmarks/micro_benchmarks/__init__.py

```diff
@@ -10,8 +10,9 @@
 from superbench.benchmarks.micro_benchmarks.cublas_function import CublasBenchmark
 from superbench.benchmarks.micro_benchmarks.cudnn_function import CudnnBenchmark
 from superbench.benchmarks.micro_benchmarks.gemm_flops_performance import GemmFlopsCuda
+from superbench.benchmarks.micro_benchmarks.memory_bw_performance import MemBwCuda
```

Comment:
As I mentioned in another comments, we should change all to `CudaXXXBenchmark`. Also for `GemmFlopsCuda`

Reply:
Updated for this benchmark, others will update in other pr later